<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadWriteWindow</class>
 <widget class="QDialog" name="ReadWriteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QLabel" name="ValueLabel">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="sizeLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>146</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="addressLabel">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="valueLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>146</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="sizeLabel">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="7">
      <widget class="QDialogButtonBox" name="buttonBox"/>
     </item>
     <item row="0" column="4" rowspan="4">
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>201</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
border: 2px solid grey;
border-radius: 5px;
text-align: center;
}
QProgressBar:horizontal {
border: 2px solid gray;
border-radius: 3px;
background: white;
}
QProgressBar::chunk:horizontal {
background-color: qlineargradient(x0: 0, x2: 1, stop: 0 #05B8CC, stop: 1 #05B8CC)
}
QProgressBar::chunk {
background-color: #05B8CC;
margin: 0.5px;
border-radius: 5px;
}
QProgressBar::chunk:vertical {
background-color: qlineargradient(y0: 1, y2: 0, stop: 1 #05B8CC, stop: 0 #05B8CC);
}
QProgressBar:vertical {
border: 2px solid gray;
border-radius: 3px;
background: white;
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="addressSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>146</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <widget class="QComboBox" name="valueTypeComboBox">
       <item>
        <property name="text">
         <string>HEX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>INT</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
